script/build

```diff
@@ -2,18 +2,11 @@
 
 'use strict'
 
-// Run bootstrap first to ensure all the dependencies used later in this script
-// are installed.
-require('./bootstrap')
-
-// Needed so we can require src/module-cache.coffee during generateModuleCache
-require('coffee-script/register')
-require('colors')
-
 const yargs = require('yargs')
```

Comment:
thanks for your review.
Maybe I should find another way to run `build --help` without external library.
this PR will be closed